lib/cucumber/formatter/fail_fast.rb

```diff
@@ -0,0 +1,21 @@
+require 'cucumber/formatter/io'
+require 'cucumber/formatter/console'
+
+module Cucumber
+  module Formatter
+    class FailFast
+      def initialize(configuration)
+        @configuration = configuration
+      end
+
+      def after_test_case(test_case, result)
+        Cucumber.wants_to_quit = true unless result.ok? @configuration.strict?
```

Comment:
That's OK, I just remembered recently that you didn't have to use them, so I stopped using them just for the sake of novelty, really. I meant to use them in this code.